adapters/tlsio_wolfssl.c

```diff
@@ -103,6 +103,23 @@ static void* tlsio_wolfssl_CloneOption(const char* name, const void* value)
                 /*return as is*/
             }
         }
+        #ifdef INVALID_DEVID
+        else if(strcmp(name, OPTION_WOLFSSL_SET_DEVICE_ID) == 0 )
+        {
+             int* value_clone; 
+  
+             if ((value_clone = (int*)malloc(sizeof(int))) == NULL) 
```

Comment:
A minor suggestion that would align with the usual coding standard we **try** to maintain: the return from master should not be cast (backed by guidance in the C99 language spec).
It would result in:
```
if ((value_clone = malloc(sizeof(int))) == NULL) 
```